update docs for team admin permissions++ (#5564)
pulumi/docs: content/docs/intro/console/accounts-and-organizations/accounts.md

obtained from the identity provider you used for signing up. For example, your
GitHub identity.

> To edit your profile information, navigate to **Profile Settings** by selecting
> on your avatar and selecting **Settings**. The **Profile** tab is displayed by default.

Reviewer: ```suggestion
To edit your profile information, navigate to **Profile Settings** by selecting
your avatar, then **Settings**. The **Profile** tab is displayed by default.
```
